lib/install.ps1

```diff
@@ -906,19 +906,13 @@ function env_add_path($manifest, $dir, $global, $arch) {
     $env_add_path = arch_specific 'env_add_path' $manifest $arch
     $dir = $dir.TrimEnd('\')
     if ($env_add_path) {
-        # GH-3785: Add path in ascending order.
-        [Array]::Reverse($env_add_path)
-        $env_add_path | Where-Object { $_ } | ForEach-Object {
-            if ($_ -eq '.') {
-                $path_dir = $dir
-            } else {
-                $path_dir = Join-Path $dir $_
-            }
-            if (!(is_in_dir $dir $path_dir)) {
-                abort "Error in manifest: env_add_path '$_' is outside the app directory."
-            }
-            Add-Path -Path $path_dir -Global:$global -Force
+        if (get_config USE_ISOLATED_PATH) {
+            Add-Path -Path '%SCOOP_PATH%' -Global:$global
+            $target_path = 'SCOOP_PATH'
+        } else {
+            $target_path = 'PATH'
         }
```

Comment:
```suggestion
        $target_path = 'PATH'
        if (get_config USE_ISOLATED_PATH) {
            Add-Path -Path '%SCOOP_PATH%' -Global:$global
            $target_path = 'SCOOP_PATH'
        }
```

Reply:
Hmm, is there any difference between these two implements?

Reply:
hoisting variable, less loc